### PR TITLE
added HomePage Tests

### PR DESCRIPTION
**Pull request by usernameNur**

### CODE REVIEW

File: src/test/java/Test/HomeTests.java [hunk 1]
```diff
@@ -0,0 +1,10 @@
+package Test;
+
+import org.testng.annotations.Test;
+
+public class HomeTests {
+    @Test
+    public void home(){
+        System.out.println("HomePage");
```
Comment (Z1zee) [inline src/test/java/Test/HomeTests.java:8]: looks good

**Review by abylkasymovE — COMMENTED** [review]
nice job

File: src/test/java/Test/HomeTests.java (hunk 1 above)
Comment (usernameNur) [reply src/test/java/Test/HomeTests.java:8]: thanks

**Review by abylkasymovE — APPROVED** [review]

**Review by Z1zee — APPROVED** [review]

**Comment by sh1mbergenov** [comment]
good!

**Pull request merged**